docs/wiki/03_HELP_AND_TROUBLESHOOTING/README.md

```diff
@@ -1 +1,38 @@
-General help with modules, instruments, install... resources, contacts, FAQ ...
\ No newline at end of file
+# Help and Troubleshooting
```

Comment:
this entire text should be in the installation troubleshooting as it pertains to the values configured during installation